Assignment2/eduardogeuens-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,56 @@
+{
+	"@context": {
+		"re": "http://assignment2b.com#",
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+		"Class01": "re:Class01",
+		"Sensor029": "Class01:Sensor029",
+		"Computer101": "Class01:Computer101",
+		"Measurement8401": "Sensor029:Measurement8401",
+		"User10A": "Computer101:User10A",

```

Comment:
You are defining them as "@type" on your data, thus you don't need to define them here.